web_src/js/svg.js

```diff
@@ -91,6 +93,8 @@ const svgs = {
   'octicon-tag': octiconTag,
   'octicon-git-branch': octiconGitBranch,
   'octicon-rss': octiconRss,
+  'octicon-checkbox': octiconCheckbox,
+  'gitea-empty-checkbox': giteaEmptyCheckbox,
```

Comment:
Oh, I see that the alphabetic sorting of this list has been broken.
Would you care to restore it?

Reply:
done (sorted by the second word)

Reply:
Thanks :+1: